samples/csharp_dotnetcore/16.proactive-messages/ProactiveAccessors.cs

```diff
@@ -6,20 +6,29 @@
 namespace Microsoft.BotBuilderSamples
 {
     /// <summary>
-    /// Defines <see cref="IStatePropertyAccessor{T}"/> for use with this bot.
+    /// This class is created as a Singleton and passed into the IBot-derived constructor.
+    ///  - See <see cref="ProactiveBot"/> constructor for how that is injected.
+    ///  - See the Startup.cs file for more details on creating the Singleton that gets
+    ///    injected into the constructor.
     /// </summary>
     public class ProactiveAccessors
```

Comment:
>ProactiveAccessors [](start = 17, length = 18)

Can you get rid of this class and register instead the JobState class as singleton in Startup.cs, and create the Accessor in the bot?  Sorry for the change, but I think this will match Jonathan's sample.